### PR TITLE
feat: lesson06

### PR DESCRIPTION
**Pull request by AyakaOnozuka131**
# [Issue No.6](https://github.com/kenmori/handsonFrontend/blob/master/work/markup/1.md#6)

**課題内容**
5で作ったものを3秒後に解決されるようにしてください

**Lesson5 contents↓**
4のこちらをつかって、こんどはPromiseオブジェクトを使って解決された値として受け取り、
`[{to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}]`
同じように
```
<ul>
 <li><a href="/bookmark.html"><img src="1.png" alt="画像1">ブックマーク</a></li>
 <li><a href="/message.html"><img src="2.png" alt="画像2">メッセージ</a></li>
</ul>
```
こちらの出力になるようにしてください

## [CodeSandBox](https://codesandbox.io/s/lesson06-f6p77) 
:Latest commit [7a2a0f7](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/6/commits/7a2a0f70a24f3dd8d2db31094b93e144f3c6f006)

## 実装上、不安なところ、重点的に見てもらいたいところ、聞きたいこと

### CODE REVIEW

File: lesson06/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    
+        a.href = `/${val[i].to}`;
+        a.textContent = `${val[i].text}`;
+        img.src = `${val[i].img}`;
+        img.alt = `${val[i].alt}`;
+    
+        fragment.appendChild(li).appendChild(a).insertAdjacentElement('afterbegin', img);
+    }
+
+    // ulリスト内にfragmentを追加する
```
Comment (taka0209) [inline lesson06/index.js]: nits,
I think this comment is not necessary because we can understand when we read the code.

File: lesson06/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}).then((val) => {
+    for (let i = 0; i < val.length; i++) {
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        const img = document.createElement('img');
+    
+        a.href = `/${val[i].to}`;
+        a.textContent = `${val[i].text}`;
+        img.src = `${val[i].img}`;
+        img.alt = `${val[i].alt}`;
```
Comment (haru-programming) [inline lesson06/index.js]: question

I felt that the code below 'a.textcontent'  did not need the template literal.
If you have a reason why they are needed, I'd love to hear it!

I wondered if I could do this↓
```  
    a.textContent = val[i].text;
    img.src = val[i].img;
    img.alt = val[i].alt;
```

File: lesson06/index.js (hunk 1 above)
Comment (AyakaOnozuka131) [reply lesson06/index.js]: Thank you for review!
Comment out removed.
[ba7eeca](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/6/commits/ba7eeca7dcccedfaf2248ab86e85add698e54650)
[CodeSandBox](https://codesandbox.io/s/lesson06-f6p77)

File: lesson06/index.js (hunk 2 above)
Comment (AyakaOnozuka131) [reply lesson06/index.js]: Thank you for review!
As you said, the template literal was unnecessary, so I removed it.
[a382856](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/6/commits/a382856b9f9bba1a3380b59dff80b1fc84d277bc)

**Review by haru-programming — APPROVED** [review]

File: lesson06/index.js (hunk 2 above)
Comment (AyakaOnozuka131) [reply lesson06/index.js]: I'm sorry after I received your approval.
Added a href template literal.
[7a2a0f7](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/6/commits/7a2a0f70a24f3dd8d2db31094b93e144f3c6f006)
[CodeSandBox](https://codesandbox.io/s/lesson06-f6p77)

**Review by taka0209 — APPROVED** [review]

**Pull request merged**